src/peft/utils/other.py

```diff
@@ -67,6 +72,8 @@ def infer_device() -> str:
         return "cuda"
     elif hasattr(torch.backends, "mps") and torch.backends.mps.is_available():
         return "mps"
+    elif is_mlu_available and is_mlu_available():
```

Comment:
I think this is a bit awkward (checking bool on what could be a function). I assume that the outcome of `is_mlu_available()` is static within a process, so we would do something like:

```python
if version.parse(accelerate.__version__) >= version.parse("0.29.0"):
    from accelerate.utils import is_mlu_available
    mlu_available = is_mlu_available()
else:
    mlu_available = False
```

WDYT?

Reply:
Great! It has been updated.